### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [Capchdo/blog#2] [General]
# index

关于 HaoBIT 等。

https://blog.haobit.top/

**Comment by CZLeader** [comment]
<button onclick="alert('缓存清理完成')">缓存清理</button>

**Comment by YDX-2147483647** [comment]
另请参考：[相关项目 · YDX-2147483647/bulletin-issues-transferred · Discussion #50](https://github.com/YDX-2147483647/bulletin-issues-transferred/discussions/50)。
